server/lambda/layer/common/nodejs/custom_modules/send-to-sbc.js

```diff
@@ -27,32 +61,56 @@ const getUnsuccessfulSbcTransactions = async (collection, arrivalKey) => {
 
 // Following NoSQL recommendation, in this case, we want to store
 // BC Services transactional data on the record itself
-const updateSbcTransactions = async (collection, id, transaction) => collection.updateOne(
-  { id },
-  { $push: { serviceBCTransactions: transaction }, $set: { updatedAt: new Date().toISOString() } },
-);
+const updateSbcTransactions = async (collection, id, transaction) =>
+  collection.updateOne(
+    { id },
+    {
+      $push: { serviceBCTransactions: transaction },
+      $set: { updatedAt: new Date().toISOString() },
+    }
+  );
 
 const postToSbcAndUpdateDb = async (collection, submission) => {
   const transaction = await postServiceItem(submission);
   await updateSbcTransactions(collection, submission.id, transaction);
   return { id: submission.id, status: transaction.status };
 };
 
-const cleanJoinArray = (a) => a
-  .filter((i) => ['string', 'number'].includes(typeof i))
-  .map((i) => String(i).trim())
-  .filter((i) => i !== '')
-  .join(', ');
+const cleanJoinArray = (a) =>
+  a
```

Comment:
do we want this style change?

Reply:
David and I chatted - we think yes, just keep this for consistency with the prettier settings. Probably a good cue to us that in general we want to use more descriptive variable names even for very simple functions, though! I wouldn't worry about changing this as part of this PR

Reply:
Problem is that no prettier config file is available in repo. So, my VS code formatted with my default prettier setting.